content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -222,9 +222,19 @@ kubectl describe pvc <pvc_name>
 ```
 
 If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+FlexVolumes allow resize if the driver is set with the `RequiresFSResize` capability to true. Since this is a alpha feature
+both  `ExpandInUsePersistentVolumes` and `ExpandPersistentVolumes` feature gates must be enabled.
 
 #### Resizing an in-use PersistentVolumeClaim
```

Comment:
@brahmaroutu Could you provide an example of how to do this with `kubectl` or an example manifest please? At present I think I understand the feature, but I'd be hard pressed to do so as a cluster operator.

Reply:
@zparnold The feature is not going into 1.12 so this PR can move to the next release. Sorry for the late notice. I will add the example ASAP.

Reply:
It is not possible from command line.